kebechet/services.py

```diff
@@ -55,6 +55,14 @@ def _encode_fslash(slug: str):
             "auth": {"header": "Private-Token", "value": "{token}"},
             "slug_method": _encode_fslash.__func__,
         },
+        "gitlabcee": {
+            "token": "KEBECHET_GITLABCEE_TOKEN",
+            "service_type": ServiceType.GITLAB,
+            "config_url": "https://gitlab.cee.redhat.com/api/v4/projects/{slug}/repository/files/\
+.thoth.yaml/raw?ref={branch}",
+            "auth": {"header": "Private-Token", "value": "{token}"},
+            "slug_method": _encode_fslash.__func__,
+        }
```

Comment:
This look good for now.  If others want to use this in the future we should have some way of adding configuration which allows them to add to this structure.